### PR TITLE
gnomeExtension.disable-unredirect: init at unstable-2021-01-17

### PR DESCRIPTION
**Pull request by eduardosm**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118310` at c443b363 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>gnomeExtensions.disable-unredirect</li>
  </ul>
</details>
<details>
  <summary>4 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `gnome3`.
Near pkgs/desktops/gnome-3/extensions/disable-unredirect/default.nix:1:39:
```
  |
1 | { lib, stdenv, fetchFromGitHub, glib, gnome3, gettext }:
  |                                       ^
```
</li>
    <li>warning: unused-argument

Unused argument: `gettext`.
Near pkgs/desktops/gnome-3/extensions/disable-unredirect/default.nix:1:47:
```
  |
1 | { lib, stdenv, fetchFromGitHub, glib, gnome3, gettext }:
  |                                               ^
```
</li>
    <li>warning: unused-argument

Unused argument: `glib`.
Near pkgs/desktops/gnome-3/extensions/disable-unredirect/default.nix:1:33:
```
  |
1 | { lib, stdenv, fetchFromGitHub, glib, gnome3, gettext }:
  |                                 ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/desktops/gnome-3/extensions/disable-unredirect/default.nix:16:3:
```
   |
16 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

**Comment by eduardosm** [comment]
> 4 suggestions:

Fixed

File: pkgs/desktops/gnome-3/extensions/disable-unredirect/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-disable-unredirect";
+  version = "2021-01-17";
```
Comment (SuperSandro2000) [inline pkgs/desktops/gnome-3/extensions/disable-unredirect/default.nix:5]:
```suggestion
  version = "unstable-2021-01-17";
```

Comment (eduardosm) [reply pkgs/desktops/gnome-3/extensions/disable-unredirect/default.nix]: Fixed

**Pull request merged**